app/assets/javascripts/components/story/StoryAttachment.js

```diff
@@ -42,7 +42,7 @@ class StoryAttachment extends React.Component {
     } = args;
     let files = filesModel;
 
-    if(files) {
+    if(!_.isEmpty(files)) {
```

Comment:
This won't work for an array like this `[{}]`:

<img width="115" alt="screen shot 2018-02-09 at 10 53 50 am" src="https://user-images.githubusercontent.com/4325587/36028690-1586bc10-0d87-11e8-8bfa-149090051284.png">

What do you wanna in this case? Not send at all when there's some empty item or filter to send just the ones that are not empty?

Reply:
yes, if is empty cant enter in this if, if have content so have to enter and return value

Reply:
And part interesting is that work, I test, put a console.log there and in case that have to enter work, and in cases that is null not enter